tests/test_transitions.py

```diff
@@ -26,6 +26,19 @@ def test_create_transition(client: Client, transition_type, parameters, name_and
     assert_transition(response)
 
 
+def test_get_transition(client: Client):
+    response = client.get_transition(service.create_transition_id())
+    logging.info(response)
+    assert_transition(response)
+
+
+@pytest.mark.skip(reason='DELETE does not work for the mocked API')
```

Comment:
Is this a limitation of Prism?

Reply:
I don't think so, since @hexjelly was able to test this in JS-sdk. But there is something deep down in the requests-library that goes wrong when sending a DELETE request to the mocked API...

Reply:
Then, I suggest creating an issue and ignoring it for this PR.